src/pat/inject/inject.js

```diff
@@ -640,21 +639,41 @@ const inject = {
                 "It seems, the server is down. Please make a screenshot and contact support. Thank you!";
         }
 
+        let fallback;
         const url_params = new URLSearchParams(window.location.search);
+        if (url_params.get("pat-inject-errorhandler.off") === null) {
+            // Prepare a error page to be injected into the document.
+
+            // Check if a valid error page can be found in the response.
+            const valid_error_page_selector = document
+                .querySelector(`meta[name=pat-inject-valid-error-page]`)
```

Comment:
This introduces a new error handling meta tag configuration parameter: ``meta[name=pat-inject-valid-error-page]``

Reply:
Why do we need this parameter? I imagine it is to exclude a case where some HTML is returned in case of an error, but we don't want to use it. What would such a case look like? Is it not reasonable to assume that any HTML that is returned with an error status is indeed an error page?

Reply:
If an error happens, it can be
- just a one liner text string ("Error") because a programmer just returned a string :)
- an html page generated by some backend service like solr, haproxy, nginx, varnish (The famous Guru meditation error)
- an error page from plone that is unstyled

In all these cases we do not have a markup structure that has a valid source for injection and hence it would fail to inject the error. We want to be able to detect that and fall back to a standard error page - but inject, if the error is usable.

Reply:
> * just a one liner text string ("Error") because a programmer just returned a string :)

No, not really. If the programmer lets a view return a string, the merit can be debated, but then that's the success case. If an error happens, you get a traceback wrapped in the Plone error page. To prevent that, the programmer would have to catch any and all errors and explicitly suppress the exception traceback, returning a string instead. I doubt that it makes sense to consider that a possible case.

> * an html page generated by some backend service like solr, haproxy, nginx, varnish (The famous Guru meditation error)

Solr responses are never directly handed to the client. The rest sounds plausible.

> * an error page from plone that is unstyled
> 
> In all these cases we do not have a markup structure that has a valid source for injection and hence it would fail to inject the error.

I don't know of any markup that could not in principle be injected. Could it be that your concern is that we get an unstyled error, i.e. the injection would not technically fail but the result would be visually “wrong” in some way?